src/Frontend/Themes/Fork/Core/Layout/Sass/_bootstrap-variables.scss

```diff
@@ -51,6 +51,18 @@ $green:         #2C9F42;
 $teal:          #20c997;
 $cyan:          #5bc0de;
 
+// brands
+$brands: ();
+// stylelint-disable-next-line scss/dollar-variable-default
+$brands: map-merge(
+    (
+      "facebook": #3b5998,
+      "twitter":  #00acee,
+      "linked-in": #0072b1
```

Comment:
Shouldn't this be `#0077b5`

Reply:
That color I found on google, when I look back there are 9999 different hex values on google for LinkedIn. I updated the color to the one I found on there website that they are using to.